### ISSUE TITLE
Increase color contrast ratio and font size

### ISSUE THREAD

**Issue by ImedAdel** [exercism/exercism#4924]
I would like to suggest changing the text `color` from `#555` to `#222`, the `font-size` from `15px` to `16px`, and the `line-height` from `21px` to `1.5`.

The low font size and low color contrast are a major barrier for me for using the website.

**Comment by ImedAdel** [comment]
Hello World, Go Track

![2019-07-06 18_11_46-Go _ Hello World _ Exercism](https://user-images.githubusercontent.com/10099857/60758630-93742600-a019-11e9-8329-5998cbb440c1.png)
---
https://exercism.io/my/tracks/go

![2019-07-06 18_10_42-Go Track _ Exercism](https://user-images.githubusercontent.com/10099857/60758631-93742600-a019-11e9-84cc-110f7f5d9fa2.png)
---
https://exercism.io/tracks/go

![2019-07-06 18_09_47-Go _ Exercism](https://user-images.githubusercontent.com/10099857/60758632-940cbc80-a019-11e9-8ec3-d0aad7de296c.png)

**Comment by iHiD** [comment]
Thanks for the examples. 

So `#555` on `#fff` at `14px regular` is has a contrast ratio of 7.46 on WCAG 2.0 AAA, so this wouldn't have flagged up as something we need to change.

<img width="647" alt="Screenshot 2019-07-06 at 17 45 20" src="https://user-images.githubusercontent.com/286476/60759000-04194380-a016-11e9-8071-37e4e1f1858c.png">

Could you explain a little more about what specifically is hard for you to see about this, and what guidelines we can use to understand this, so we can work out how we can improve things. Thanks.

**Comment by ImedAdel** [comment]
Hello Jeremy :), it's not really because the colors didn't pass a specific test but rather that subjectively I find the text quite hard to read (and I'm wearing my glasses).

**Comment by iHiD** [comment]
Hey Imed :) So I really care about accessibility, but it's also really hard when I don't have issues with contrast etc to know what is good/bad. We use WCAG recommendations, and change anything where we don't meet those recommendations, but beyond that it's quite hard to know what we can do.

For example, the first screenshot you posted is actually #333, which is pretty dark. The font does look very thin/light on your screenshot though, compared to how it looks for me. (Screenshot below). What happens if you increase the magnification in your browser? Does the font-size go up (second screenshot has it a couple of sizes bigger in Chrome on my machine)?

<img width="566" alt="Screenshot 2019-07-06 at 19 08 37" src="https://user-images.githubusercontent.com/286476/60759723-a4c13080-a021-11e9-8e37-d5efd69beba3.png">
<img width="1246" alt="Screenshot 2019-07-06 at 19 09 47" src="https://user-images.githubusercontent.com/286476/60759724-a559c700-a021-11e9-9545-e65a31775c84.png">

If you let me know which OS/browser you're using, I'll try and debug a little more for you. I'm off now for two weeks but will check back when I get back :)

**Comment by ImedAdel** [comment]
Your screenshot is definitely legible for me! I'm using Chrome 75 on Windows 10. 

The first is one is more legible than the others, which use `#555`

**Comment by PatrickMcSweeny** [comment]
Using relative units for font sizes would help users who have increased the default font size in their browser.

**Comment by PatrickMcSweeny** [comment]
I agree that the existing font sizes are unnecessarily small and I mentioned that in #4521

**Comment by sshine** [comment]
@ImedAdel wrote:
> I would like to suggest changing the text `color` from `#555` to `#222`, the `font-size` from `15px` to `16px`, and the `line-height` from `21px` to `1.5`.

@iHiD wrote:
> the first screenshot you posted is actually `#333`, which is pretty dark.

It seems that there is some discrepancy here that I need to understand: Does the website render with a different color with @ImedAdel's browser and monitor than with @iHiD's? The text in @ImedAdel's screenshot, when I go to that page, is `#333` here also. Maybe we could clarify this first.

Perhaps, if the explanation that @yyyc514 suggests is *a font stack issue*, the first step would be to ensure that the font renders predictably before changing colors. The font information I get when rendering the page (Linux, Chrome 76.0.3809.100) looks like this:

```css
font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
  Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', Helvetica, Arial, sans-serif,
  "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

@ImedAdel: It would be most helpful if you could help explain how the font/color got unreadable to you.

On a similar note, the gray text below (`#999`) currently fails [WCAG 2.0 contrast check](https://accessible-colors.com/).

This could be fixed by changing `#999` to `#666` or lower.

![aaa](https://user-images.githubusercontent.com/50879/65428054-ef2d9e80-de13-11e9-80db-a431f9dbaeba.png)

![aaa-success](https://user-images.githubusercontent.com/50879/65428174-3a47b180-de14-11e9-804b-421ea6baa130.png)

**Comment by iHiD** [comment]
> The font information I get when rendering the page

For those who don't read CSS, that font-stack means that the browser is effectively using the default font on your browser/OS. For Mac, that is San Fransisco, which looks lovely to my eyes. For Windows it seems to be something else that looks less lovely, but which might look lovely to Windows users' eyes. For Linux, it might be one of the options listed (eg Ubuntu) or something else.

**Comment by yyyc514** [comment]
> but which might look lovely to Windows users' eyes

It doesn't (as a former Windows user), but it's possible it's improved in the past 10 years - but from looking at the captures it doesn't seem to have. :p-)

**Comment by ErikSchierboom** [comment]
Hello 👋

With the launch of Exercism v3, we are **closing all issues in this repository** to help give us a clean slate to detect new problems. If this issue is still relevant to Exercism v3 (e.g. it's a feature that we haven't implemented in v3, or a bug that still exists), please reopen it and we will review it and post an update on it as soon as we get chance.

Thanks for helping make Exercism better, and we hope you enjoy v3 🙂
